src/MGA/Check/UnreachablePath.php

```diff
@@ -46,13 +49,29 @@ class UnreachablePath
         '.git/',
         '.git/refs/',
         '.gitignore',
+        '.idea',
         '.hg/',
+        '.htaccess.sample',
         '.svn/',
         'app/etc/enterprise.xml',
         'chive',
         'composer.lock',
+        'cron.sh',
+        'index.php.sample',
         'info.php',
+        'install.php',
+        'LICENSE.html',
+        'LICENSE.txt',
+        'LICENSE_AFL.txt',
+        'LICENSE_EE.html',
+        'LICENSE_EE.txt',
+        'mage',
```

Comment:
Is there anything interesting in these files?  We're already using another method to detect enterprise vs community